src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -139,6 +142,46 @@ class ObservationStore(
         }
   }
 
+  fun fetchStartableObservations(
+      plantingSiteId: PlantingSiteId? = null
+  ): List<ExistingObservationModel> {
+    val maxStartDate = LocalDate.ofInstant(clock.instant(), ZoneOffset.UTC).plusDays(1)
```

Comment:
Curious why the +1 day?

Reply:
For example, say we have a planting site in east Asia, in time zone UTC+8:00, with an observation that starts May 1. We will actually need to start the observation at 16:00 UTC on April 30. But we won't know the time offset until after we've queried it from the database on April 30. So the query needs to include dates that look like "tomorrow" according to the server's clock.

Reply:
Ah, got it. Thanks for the explanation.